packages/icons/src/library/archive.js

```diff
@@ -5,7 +5,11 @@ import { SVG, Path } from '@wordpress/primitives';
 
 const archive = (
 	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-		<Path d="M19 6.2h-5.9l-.6-1.1c-.3-.7-1-1.1-1.8-1.1H5c-1.1 0-2 .9-2 2v11.8c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V8.2c0-1.1-.9-2-2-2zm.5 11.6c0 .3-.2.5-.5.5H5c-.3 0-.5-.2-.5-.5V6c0-.3.2-.5.5-.5h5.8c.2 0 .4.1.4.3l1 2H19c.3 0 .5.2.5.5v9.5zM8 12.8h8v-1.5H8v1.5zm0 3h8v-1.5H8v1.5z" />
+		<Path
+			fillRule="evenodd"
```

Comment:
I'm always curious when these clip and fill-rules are added, in some cases they are necessary. In this case, since they weren't there before, does it work without them?